hack/local-up-cluster.sh

```diff
@@ -681,6 +716,9 @@ EOF
 fi
 }
 
+# let's check that etcd is in path before moving on
```

Comment:
`kube::etcd::start` basically does this check, and also verifies that it's the correct version. do we need to duplicate this check?